docs/user-guide/vpcs.md

```diff
@@ -14,6 +14,7 @@ metadata:
 spec:
   ipv4Namespace: default # Limits which subnets can the VPC use to guarantee non-overlapping IPv4 ranges
   vlanNamespace: default # Limits which Vlan Ids can the VPC use to guarantee non-overlapping VLANs
+  mode: l2vni  # mode defaults to l2vni. Other option is l3vni
```

Comment:
the default VPC mode is "" (empty string) and it's a l2vni for backwards compat.